app/src/main/java/is/xyz/mpv/MPVActivity.kt

```diff
@@ -1818,17 +1850,20 @@ class MPVActivity : AppCompatActivity(), MPVLib.EventObserver, TouchGesturesObse
         eventUiHandler.post { eventUi(eventId) }
     }
 
-    // Gesture handler
+// Gesture handler
 
     private var initialSeek = 0f
     private var initialBright = 0f
     private var initialVolume = 0
     private var maxVolume = 0
+
     /** 0 = initial, 1 = paused, 2 = was already paused */
     private var pausedForSeek = 0
 
     private fun fadeGestureText() {
         fadeHandler.removeCallbacks(fadeRunnable3)
+        // No need to show OSD
+        binding.controls.visibility = View.GONE
```

Comment:
this should call one of the existing functions due to separation of concerns.
but why do this anyway?

Reply:
to hide the controls during gesture seek/forward etc
although it's not a good way, might as well skip it for now idk.